src/org/labkey/test/util/exp/DataClassAPIHelper.java

```diff
@@ -0,0 +1,74 @@
+package org.labkey.test.util.exp;
+
+import org.labkey.remoteapi.CommandException;
+import org.labkey.test.params.FieldDefinition;
+import org.labkey.test.params.experiment.DataClassDefinition;
+import org.labkey.test.params.experiment.SampleTypeDefinition;
+import org.labkey.test.util.TestDataGenerator;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class DataClassAPIHelper
+{
+
+
+    /**
+     * Create a dataclass in the specified container with the fields provided.
+     *
+     * @param containerPath Container in which to create the dataclass.
+     * @param dataClassDefinition domain properties for the new dataclass.
+     * @return A TestDataGenerator for inserting rows into the created dataclass.
+     */
+    static public TestDataGenerator createEmptyDataClass(String containerPath, DataClassDefinition dataClassDefinition)
+    {
+        deleteDomain(new FieldDefinition.LookupInfo(containerPath, "exp.data", dataClassDefinition.getName()));
+
+        TestDataGenerator dgen;
+        try
+        {
+            dgen = TestDataGenerator.createDomain(containerPath, dataClassDefinition);
+        }
+        catch (CommandException e)
+        {
+            throw new RuntimeException("Failed to create sample type.", e);
+        }
+        return dgen;
```

Comment:
Why does this declare and return `dgen` outside of the try/catch?
```suggestion
        try
        {
            return TestDataGenerator.createDomain(containerPath, dataClassDefinition);
        }
        catch (CommandException e)
        {
            throw new RuntimeException("Failed to create sample type.", e);
        }
```